nodejs/nodejs-cloud-run-hello-world/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use the official lightweight Node.js 12 image.
+# https://hub.docker.com/_/node
+FROM node:12-slim
+
+# Create and change to the app directory.
+WORKDIR /usr/src/app
+
+# Copy application dependency manifests to the container image.
```

Comment:
In other apps we are copying the source and building final container image - all in docker file, so it's self contained.

Reply:
That's what's happening here.